.kitchen.yml

```diff
@@ -112,9 +97,27 @@ provisioner:
           renew-by-default: true
 
 suites:
+  - name: default
+    provisioner:
+      pillars:
+        top.sls:
+          base:
+            '*':
+              - letsencrypt
+        letsencrypt.sls:
+          letsencrypt:
+            config: |
+              server = https://acme-v01.api.letsencrypt.org/directory
```

Comment:
In `kitchen.yml`, shouldn't this point to LetsEncrypt's staging server, in order to play safe in case someone tries to test the `domains` state?

```yaml
server = https://acme-staging.api.letsencrypt.org/directory
```

Reply:
great - changed it, thx